DESCRIPTION

```diff
@@ -20,7 +20,7 @@ URL: https://pharmaverse.github.io/admiraltemplate,
 Depends: 
     R (>= 4.1)
 Imports:
-    admiral (>= 1.0.0),
+    admiral (>= 1.1.0),
```

Comment:
Actually admiral was patched after the 1.1 release (see [here](https://github.com/pharmaverse/admiral/releases)) so we should point to that.